lifterlms/courses-lessons/exclude-posts-from-user-roles.php

```diff
@@ -0,0 +1,52 @@
+/**
+ * Exclude posts (given their IDs) from being seen by users given a list of roles.
+ * 
+ * Learn more at: https://lifterlms.com/link-to-content-if-available-or-remove-this-line/
+ * 
+ * You can add this recipe to your site by creating a custom plugin
+ * or using the Code Snippets plugin available for free in the WordPress repository.
+ * Read this companion documentation for step-by-step directions on either method.
+ * https://lifterlms.com/docs/adding-custom-code/
+ */
+
+function exclude_posts_for_roles($query) {
```

Comment:
We should add spacing before/after brackets to follow our own guidelines and WP standards throughout the whole file, for example:

```
function exclude_posts_for_roles( $query ) {
   if ( is_admin() ) {
       return;
   }
   // ...
```